### DISCUSSION TITLE
html tags altered in quarto (and in Rstudio visual editor)

### DISCUSSION THREAD

**Discussion by dkpalmers** [quarto-dev/quarto-cli#917] [Q&A]
I have a page originally written in Rmarkdown that allows users to hover over text and get additional explanation. I have started a new project and want to re-use some of the code. However, when I render the same code in quarto, or in an Rstudio project when I change from viewing "source" to "visual editor" and knit, the existing code is changed and losses functionality.

Here is a link to the original working code and the altered code, as well as the resulting html file and the styles.css file. https://github.com/byuistats/Math326_Quarto4/tree/master/docs/issue_files

Please help me know how to keep using this hovering ability and still use quarto.

**Comment by jjallaire** [comment]
In terms of the visual editor, there is not currently a way to prevent it from converting raw divs and spans to pandoc divs and spans. If that's a problem I think best to keep this out of the visual editor.

In terms of mosaic functionality, I think there is some extra processing going on that I don't fully understand (i.e. the `showhide() call has to have some Javascript it's calling somewhere, but I can't see where that would be -- perhaps something injected by `mosaic`?). I would follow up with the authors of the mosaic package on this.
